app/views/users/_unfollow.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_for @unfollow, html: {method: :delete}, remote: true do |f| %>
```

Comment:
Nó báo lỗi First argument in form cannot contain nil or be empty. Cho mình hỏi tại sao lại thế?